docs/source/development-guide/style-guide/naming-conventions.rst

```diff
@@ -31,46 +30,49 @@ The general filename convention is as follows::
   * SWE: ``l0``, ``l1a``, ``l1b``, ``l2``, ``l3``
 
 * ``<descriptor>`` stores information specific to the instrument and can store any information that is relevant. For
-  example, it will typically contain the data product name, and additionally may contain the pointing number (e.g.
-  ``burst-7`` for 'burst' mode and the 7th repointing). This field can have any text or numbers as long as it doesn't
-  include underscores. The following is a list of expected descriptors for each instrument:
+  example, it will typically contain the data product name, the sensor name, the timespan of the data, etc. This field
+  can have any text or numbers as long as it doesn't include underscores. The following is a list of expected
+  descriptors for each instrument:
 
   * CoDICE: ``hskp``, ``eng``, ``lo-priority``, ``lo-sw-angular``, ``lo-nsw-angular``, ``lo-pha``, ``hi-pha``,
     ``lo-sw-species``, ``lo-nsw-species``, ``hi-omni``, ``hi-sectored``
-  * GLOWS: ``histogram``, ``de``, ``<pointing number>``
+  * GLOWS: ``histogram``, ``de``
   * HIT: TBD
   * IDEX: TBD
-  * IMAP-Hi: TBD
+  * IMAP-Hi: ``45sensor-counts``, ``90sensor-counts``, ``45sensor-flux-3month-map`` (more TBD)
```

Comment:
I forgot to bring this up on the call today, but for consistency should we put `3month` at the end?
`flux-map-3month` similar to how SWAPI  has `sci-1min` with the frequency last.
cc @tech3371